src/utils/httpUtils.ts

```diff
@@ -11,3 +13,7 @@ export function getURL(baseURL: string, path: string, orgId?: string): string {
 function routeToGateway(baseURL: string): boolean {
   return baseURL.includes('snykgov.io');
 }
+
+function isValidOrg(orgId?: string): boolean {
+  return Boolean(orgId && ORG_ID_REGEXP.exec(orgId));
```

Comment:
q: Why the boolean constructor here?

Reply:
Do you prefer casting the value to boolean with `!!`?

Reply:
Why would we need to cast it at all? Shouldn't we just be able to do (roughly):

```
return orgId !== undefined && ORG_ID_REGEXP.test(orgId)
```

Reply:
You are right, since orgId can be undefined or a string. I just thought it is safer to ensure that orgId has a value, and is not null before doing the regex check. But that should not be a problem when using typescript.

Reply:
You can also do the following, if you want to catch the case. But if that is a possible situation, you should probably update the type definitions too :)

```
return orgId != null && ORG_ID_REGEXP.test(orgId)
```